Include/Tests/TestUtility.hpp

```diff
@@ -0,0 +1,12 @@
+#ifndef ASHKAL_TEST_UTILITY_HPP
+#define ASHKAL_TEST_UTILITY_HPP
+#include "Ashkal/Ashkal/Matrix.hpp"
+
+namespace Ashkal {
+
+  bool equals(const Matrix& a, const Matrix& b, int sig_figures = 14);
```

Comment:
The default should be 15, but your test will be:

|a - b| <= (sig_figures - 1)